dbt/include/bigquery/macros/materializations/view.sql

```diff
@@ -9,9 +9,13 @@
 
 
 {% materialization view, adapter='bigquery' -%}
+    -- grab current tables grants config for comparision later on
+    {% set grant_config = config.get('grants') %}
+
     {% set to_return = create_or_replace_view() %}
 
     {% set target_relation = this.incorporate(type='view') %}
+    {% do apply_grants(target_relation, grant_config) %}
```

Comment:
We don't need this here, since the grant is already happening within `create_or_replace_view()`